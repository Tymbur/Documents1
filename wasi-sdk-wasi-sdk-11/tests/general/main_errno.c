#include <stdio.h>
#include <errno.h>
#include <string.h>

// It isn't required that errno be zero on entry to main, but
// for tidiness' sake, if we ever do things during startup that
// do set errno, we should reset it for tidiness' sake.
int main(void) {
    int n = errno;
    printf("initial errno is %d: %s\n", n, strerror(n));
    return 0;
}
