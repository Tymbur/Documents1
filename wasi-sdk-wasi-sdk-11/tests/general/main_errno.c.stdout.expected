initial errno is 0: Success
